src/app/components/feed/Feed.js

```diff
@@ -38,26 +35,16 @@ export const FeedComponent = ({ routeParams, ...props }) => {
   }
 
   const commonSearchProps = {
-    searchUrlPrefix: `/${routeParams.team}/feed/${feed.dbid}/${tab}`,
+    searchUrlPrefix: `/${routeParams.team}/feed/${feed.dbid}/`,
```

Comment:
suggestion: Can probably remove the slash at the end, to keep the url clean

Reply:
Yes, @danielevalverde plz remove the trailing slash. Also to avoid double slashes when that prefix is concatenated with any suffixes in `SearchResults.js`